prx/rc1/multiplayer/Packet.h

```diff
@@ -203,6 +203,8 @@ typedef struct {
     float size;
     char text[50];
 } MPPacketSetHUDText;
+static u16 game_state_flags = 0xFF;
```

Comment:
Can you do it without adding global variables? Make them `#define`s if you just need the values.

Reply:
sure, it would make more sense to define them locally anyways.